src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1001,4 +1001,72 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.ExpressionStatement
         End Function
     End Class
+
+    Partial Class BoundAddRemoveHandlerStatement
+        Implements IEventAssignmentExpression
```

Comment:
>...to introduce a wrapper node around this node to act as the **expression**.

You mean statement, right? If so, I had the something in mind, but how do you add this wrapper node to its enclosing statement? e.g. a BlockStatement.